rs/bitcoin/ckbtc/minter/src/metrics.rs

```diff
@@ -1,9 +1,63 @@
 use crate::state;
-use std::cell::Cell;
+use std::cell::{Cell, RefCell};
+use std::collections::BTreeMap;
+use std::io::Error;
+use std::time::Duration;
 
 thread_local! {
     pub static GET_UTXOS_CLIENT_CALLS: Cell<u64> = Cell::default();
     pub static GET_UTXOS_MINTER_CALLS: Cell<u64> = Cell::default();
+    pub static UPDATE_CALL_LATENCY: RefCell<BTreeMap<usize,LatencyHistogram>> = RefCell::default();
+}
+
+pub(crate) const LATENCY_HISTOGRAM_NUM_BUCKETS: usize = 11;
+pub(crate) const LATENCY_HISTOGRAM_BUCKET_SIZE_MS: u64 = 500;
```

Comment:
now as a follow-up I think we could cover more data with less buckets by having adaptive ranges, typically a power of 2:

```rust
const BUCKETS_MS: [u64; 10] = [500, 1_000, 2_000, 4_000, 8_000, 16_000, 32_000]
```
This would use in total 8 buckets (including the "infinity" bucket). This could be done here or as a separate PR. We may refine those buckets later once we have actual data to act upon.